### DISCUSSION TITLE
Is there a tool to transcribe video assets?

### DISCUSSION THREAD

**Discussion by brianburks** [pimcore/pimcore#11767] [Q&A / Support] [answered]
I wish to use DAM for church assets.  A major use will be to store videos of sermons.  I have two questions: 1) Is there an extension that will automatically transcribe video to text and save it as metadata (approx 3,000 words per file).  2) Even if I have to manually enter the text as metadata, is the text indexed and searchable from the DAM metadata search bar?

**Comment by ckemptner** [comment, marked as answer]
1) I am not aware of any. You could use a cloud service such as https://cloud.google.com/speech-to-text/

2) Yes.

**Reply by brianburks** [reply]
Thanks so much for your quick reply!
